xmrstak/backend/cpu/crypto/cryptonight_aesni.h

```diff
@@ -250,15 +250,15 @@ void cn_explode_scratchpad(const __m128i* input, __m128i* output)
 		_mm_store_si128(output + i + 2, xin2);
 		_mm_store_si128(output + i + 3, xin3);
 
-		if(PREFETCH)
+		if (PREFETCH)
 			_mm_prefetch((const char*)output + i + 0, _MM_HINT_T2);
 
 		_mm_store_si128(output + i + 4, xin4);
 		_mm_store_si128(output + i + 5, xin5);
 		_mm_store_si128(output + i + 6, xin6);
 		_mm_store_si128(output + i + 7, xin7);
 
-		if(PREFETCH)
+		if (PREFETCH)
```

Comment:
Whitespace changes like that are annoying in a collaborative environment. Please remove them from your pr.

Reply:
Updated and pushed.